### ISSUE TITLE
Crashing upon entering dimension

### ISSUE THREAD

**Issue by IyaTaisho** [Sunekaer/stoneBlock#352]
## Bug Report
I was trying to create a new dimension using RF Tools Dimensions for Draconium.  I had completed creating one that had only the following:
* Draconium Ore
* Huge Orbs
* Plains biome

My dimension built fine and had power.  I teleported into it only to crash.  It maybe something to do with the attributes of the dimension just causing issues.  I'm uncertain.  However, I'm stuck there now and each time I boot up, the game loads for a few seconds, I can look down, then I crash.

From what I can tell, the dimension looked like it had red like lightning or something going on, which maybe causing serious issues.

If I can at minimum get out of the dimension, it would be great.

## Steps to Reproduce (for bugs)
1. Create an RF Tools Dimension with the above attributes.
2. Try entering the world.
## Logs
Here is my crash report: [Link](https://pastebin.com/rJ5sHsuc)

## Client Information
* Modpack Version: 1.0.20
* Server/LAN/Single Player: Single player

**Comment by knoxhack** [comment]
Reported to mod author:
https://github.com/McJtyMods/RFToolsDimensions/issues/238

**Comment by Danklor** [comment]
Taking a look at the crash dump indicates that it dies while generating chunks. One possible quick workaround is temporarily setting your video settings to have a render distance of the minimum (i think 2), teleporting back home, then restoring your video settings.  If that doesn't work you'll probably need to use an external world editor to change your position to your home.

**Comment by IyaTaisho** [comment]
Thanks for your help.  When I changed my video settings, it helped a bit.  However, I was still stuck.  I found if I killed Minecraft completely after crashing (close it out and restart Twitch), it seemed to let me have a few more seconds.  I finally managed to initiate a teleport and got home.

I immediately got rid of that dimension, lol.  However, I was able to generate a proper Draconium Ore in Orbs dimension after I rebuilt it with a Digit added.  I now have Draconium orbs everywhere in a dimension.

Thanks once more.
